VideoWeb/VideoWeb/ClientApp/src/app/shared/browser.constants.ts

```diff
@@ -0,0 +1,11 @@
+export const browsers = {
+    Firefox: 'Firefox',
+    Safari: 'Safari',
+    Chrome: 'Chrome',
+    MSEdge: 'MS-Edge',
+    MSEdgeChromium: 'MS-Edge-Chromium',
+    Samsung: 'Samsung',
+    Opera: 'Opera',
+    Brave: 'Brave',
+    MSIE: 'MSIE'
```

Comment:
Maybe make the key InternetExplorer or MSInternetExplorer just so it's a bit more obvious without having to think.